packages/common/src/templates/configuration/prettierRC/ui.tsx

```diff
@@ -152,6 +152,19 @@ export class ConfigWizard extends React.Component<ConfigurationUIProps> {
             line instead of being alone on the next line.
           </ConfigDescription>
         </PaddedConfig>
+        <PaddedConfig>
```

Comment:
Nice! I think we also need to add this to the preferences: `packages/app/src/app/pages/common/Modals/PreferencesModal/CodeFormatting/Prettier/index.js`

Reply:
Added there too :)